### ISSUE TITLE
Accessibility  issues

### ISSUE THREAD

**Issue by smeggins** [Tejaswini411/Aloha_Project#1]
List of issues:
 
Contrast issues:
All white text
The grey text used in product highlights
 
Link text:
Phone number link text 
Nav (specifically ABOUT
 
Labels:
Shopping cart icon does not have a label and can only be visually identified
Email input does not have a label and uses placeholder text to denote what it is.
 
Image alt text missing:
Logo
Shopping cart
Hero image of fashionable girl
All clothes items
